### DISCUSSION TITLE
Is there a way to kill the note like dired-do-kill-lines

### DISCUSSION THREAD

**Discussion by jayrajput** [ahyatt/ekg#69] [Q&A]
Is there a way to kill the note (not deleting). By kill, I mean to not show in the view.

**Comment by ahyatt** [comment]
There isn't a way to mark it hidden.  You can trash it, though, which will remove it from all views but not actually delete it.

How would you want it to behave if it was removed from the view?  It that just temporary or is it permanently removed?  Removed from all views or just the current one?

**Comment by jayrajput** [comment]
Yeah hidden is what I meant. I was able to code one based on `ekg-notes-delete`

```
    (defun ekg-notes-hide ()
      "Hide the current note."
      (interactive)
      (let ((inhibit-read-only t))
	(ewoc-delete ekg-notes-ewoc (ewoc-locate ekg-notes-ewoc))
	(ekg--note-highlight)))
```
My use case is to find the note. I do not remember the exact tag, so I search with a tag or multiple tags which gave me too many notes. So now I want to hide the notes which I am not interested in. I am also hopeful that hiding should have more cases. I also conveniently mapped the `ekg-notes-hide` to `k`.

**Comment by ahyatt** [comment]
Ah, interesting, yes that does make sense.  This seems worth adding.  Do you want to send a pull request (so you are recorded as the proper author), or should I just add it based on your code above?

**Reply by jayrajput** [reply]
I  will send a PR. Give me some time. Thanks.
